src/authkit-callback-route.spec.ts

```diff
@@ -0,0 +1,188 @@
+import type { LoaderFunction } from '@remix-run/node';
+import { workos as workosInstance } from '../src/workos.js';
+import { authLoader } from './authkit-callback-route.js';
+import {
+  createRequestWithSearchParams,
+  createAuthWithCodeResponse,
+  assertIsResponse,
+} from './test-utils/test-helpers.js';
+
+// Mock dependencies
+jest.mock('../src/workos.js', () => ({
+  workos: {
+    userManagement: {
+      authenticateWithCode: jest.fn(),
+      getJwksUrl: jest.fn(() => 'https://api.workos.com/sso/jwks/client_1234567890'),
+    },
+  },
+}));
+
+describe('authLoader', () => {
+  let loader: LoaderFunction;
+  let request: Request;
+  const workos = jest.mocked(workosInstance);
+
+  beforeAll(() => {
+    // Silence console.error during tests
+    jest.spyOn(console, 'error').mockImplementation(() => {});
+  });
+
+  beforeEach(async () => {
+    jest.resetAllMocks();
```

Comment:
Maybe we also set `resetMocks` in the Jest config as well? I think in other projects we tend to always set `clearMocks: true` and `resetMocks: true` as a natural default, since you almost always want to reset/clear everything between tests.